src/main/java/de/dhbw/karlsruhe/Main.java

```diff
@@ -1,8 +1,19 @@
 package de.dhbw.karlsruhe;
 
+import de.dhbw.karlsruhe.services.StartUpDialogService;
+
 public class Main {
 
   public static void main(String[] args) {
-    System.out.println("Hello world!");
+    StartUpDialogService startUpDialogService = new StartUpDialogService();
+
+    boolean successfulLogin = startUpDialogService.login();
+
+    if (successfulLogin) {
+      System.out.println("You have successfully logged in!");
+    } else {
+      System.out.println("Login failed!");
```

Comment:
Eventuell sollte diese Loginbestätigung durch den startUpDialogService ausgegeben werden und nicht in der main() Methode, da es fachlich noch dazu gehört?

Reply:
Es war eigentlich gedacht, dass dort mit dem nächsten Teil der Anwendung (bspw. Auswahlfunktion) weitergemacht wird. Sollte hier nur als "Placeholder" dienen.